### ISSUE TITLE
FIRST & LAST do not work for string values

### ISSUE THREAD

**Issue by OptrixAU** [influxdata/influxdb#6438]
### Bug report

InfluxDB 0.12.1
Ubuntu 14.04 LTS (Amazon Web Services)

__Steps to reproduce:__

1. Add some measurements that have both string and floating-point values...
`
target,id=1 position="1283.002N 0213.2293E",speed=12.02
target,id=1 position="1283.002N 0213.2293E",speed=12.05
target,id=1 position="1283.003N 0213.2294E",speed=12.14

2. Run a query to get the latest values...
SELECT LAST(speed), LAST(position) FROM target GROUP BY id
SELECT LAST(speed) FROM target GROUP BY id;
SELECT LAST(position) FROM target GROUP BY id
`

__Expected behavior:__ 
In the three queries, I'd expect to see...
a) Both the most recent speed and the most recent position
b) The most recent speed
c) The most recent position

__Actual behavior:__ 
a) The 'position' column is empty (unexpected)
b) The most recent speed (expected)
c) No rows returned (unexpected)

__Additional info:__ 

I of course understand why aggregation functions such as MEDIAN, MEAN etc. don't work on strings, as that makes no sense. Even some selectors have little practical purpose on strings. However, since there's no need to perform any actual calculations on the string values in a FIRST or LAST query, they really should work return the expected values.

**Comment by jsternberg** [comment]
Both `first()` and `last()` seem to work on strings for me.

```
> create database mydb
> use mydb
> insert target,id=1 position="1283.002N 0213.2293E",speed=12.02
> insert target,id=1 position="1283.002N 0213.2293E",speed=12.05
> insert target,id=1 position="1283.003N 0213.2294E",speed=12.14
> SELECT LAST(speed), LAST(position) FROM target GROUP BY id
name: target
tags: id=1
time    last    last_1
----    ----    ------
0       12.14   1283.003N 0213.2294E
```

**Comment by ianlutzeigen** [comment]
I had a similar issue; it would sometimes and sometimes not... i doubted my very sanity until i found how to make it work and make it not work. Your case is based on tags, so its different, but the pain i share

https://github.com/influxdata/influxdb/issues/6446

**Comment by jsternberg** [comment]
It looks like this may have been the same as #6495 and would have been fixed by #6539. If this problem still exists after upgrading to 0.13, please comment here so I can reopen the issue. Thanks.
